massa-node/base_config/testnet.toml

```diff
@@ -0,0 +1,110 @@
+version = "TEST.4.2"
```

Comment:
What is that config file for ?

Reply:
It's the config file setted for a unit test environment, I know it's a little bit dark to do this :rofl: but it will be changed soon after a work of @yvan-sraka. 

Check the lines 13 and 22 in massa-client/src/tests/scenarios.rs

Reply:
maybe hide it in a test_config folder outside of the base config then x)

Reply:
We can also create the test file at the beginning of the test and remove it at the end?!

Reply:
@yvan-sraka wooo :open_mouth:

Reply:
I just moved the test config in massa-node/src/tests with a good comment. I don't know if dumping a test file would be a better idea @yvan-sraka, may be later after more progression in the setting side?

Reply:
Yes we could wait for https://github.com/massalabs/massa/pull/1848 to that :)